data/pathfinder/paizo/roleplaying_game/core_rulebook/cr_feats.lst

```diff
@@ -166,7 +166,7 @@ Step Up											CATEGORY:FEAT		TYPE:Combat
+Stunning Fist										CATEGORY:FEAT		TYPE:Combat.AttackOption			PREABILITY:1,CATEGORY=FEAT,Improved Unarmed Strike																																																										PRESTAT:2,DEX=13,WIS=13	PRETOTALAB:8									DEFINE:StunningFistMonkLVL|0	DEFINE:StunningFistAttack|0	DEFINE:StunningFistDC|0											DESC:You know just where to strike to temporarily stun a foe. %1/day (DC %2)|StunningFistAttack|StunningFistDC																					STACK:NO																																																		ABILITY:Ability Focus|AUTOMATIC|Stunning Fist																																																																																																																																																				BONUS:VAR|StunningFistDC|2|PREABILITY:1,CATEGORY=FEAT,Ability Focus (Stunning Fist)	BONUS:VAR|StunningFistDC|10+(TL/2)+WIS	BONUS:VAR|StunningFistAttack|floor(MonkLVL+(MonkLVL/4))|TYPE=NoStack															SOURCEPAGE:p.135	BENEFIT:You must declare that you are using this feat before you make your attack roll (thus, a failed attack roll ruins the attempt). Stunning Fist forces a foe damaged by your unarmed attack to make a Fortitude saving throw (DC %1), in addition to dealing damage normally. A defender who fails this saving throw is stunned for 1 round (until just before your next turn). A stunned character drops everything held, can't take actions, loses any Dexterity bonus to AC, and takes a -2 penalty to AC. You may attempt a stunning attack once per day for every four levels you have attained (but see Special), and no more than once per round. Constructs, oozes, plants, undead, incorporeal creatures, and creatures immune to critical hits cannot be stunned.|StunningFistDC
```

Comment:
Not yet. Life is getting in the way right now. I haven't forgotten about this just haven't had time to get to it. @LegacyKing